lms/templates/courseware/info.html

```diff
@@ -49,7 +49,11 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
     <section class="updates">
       % if studio_url is not None and masquerade and masquerade.role == 'staff':
         <div class="wrap-instructor-info studio-view">
-          <a class="instructor-info-action" href="${studio_url}">${_("View Updates in Studio")}</a>
+          <a class="instructor-info-action" href="${studio_url}">
+            ${_("View Updates in {studio_name}").format(
+                studio_name=settings.STUDIO_SHORT_NAME,
```

Comment:
@singingwolfboy You need to verify this on staging; I'm not sure we set these variables in the LMS.